content/tutorials/modeling_movement/GRASS_movement.qmd

```diff
@@ -0,0 +1,565 @@
+---
+title: "Modeling Movement in GRASS"
+author: "Michael Barton & Anna Petrosova"
+date: 2025-02-10
+date-modified: today
+format: 
+  html:
+    embed-resources: true
+    toc: true
+    code-tools: true
+    code-copy: true
+    code-fold: false
+page-layout: article
+categories: [intermediate, advanced, GUI, raster, cost surface, least cost path]
+description: Generating a cumulative cost surface and least cost path with *r.walk* and *r.path* to model movement by walking across a landscape.
+engine: jupyter
+execute: 
+  eval: false
+Jupyter: python3
+
+# © Michael Barton & Anna Petrosova 2025
+---
+
+GRASS has sophisticated tools to model movement across terrain, including [***r.cost***](https://grass.osgeo.org/grass-stable/manuals/r.cost.html), [***r.walk***](https://grass.osgeo.org/grass-stable/manuals/r.walk.html), [***r.drain***](https://grass.osgeo.org/grass-stable/manuals/r.drain.html), and [***r.path***](https://grass.osgeo.org/grass-stable/manuals/r.path.html). In this tutorial, we will use ***r.walk*** and ***r.path*** to determine the best walking path to reach a destination, like a hospital.
```

Comment:
```suggestion
GRASS has sophisticated tools to model movement across terrain, including [r.cost](https://grass.osgeo.org/grass-stable/manuals/r.cost.html), [r.walk](https://grass.osgeo.org/grass-stable/manuals/r.walk.html), [r.drain](https://grass.osgeo.org/grass-stable/manuals/r.drain.html), and [r.path](https://grass.osgeo.org/grass-stable/manuals/r.path.html). In this tutorial, we will use ***r.walk*** and ***r.path*** to determine the best walking path to reach a destination, like a hospital.
```

Reply:
I think the link being green is enough as a highlight, also to be consistent with other tutorials